xonsh/completers/vox.py

```diff
@@ -0,0 +1,32 @@
+import os
+from xonsh.__amalgam__ import builtins
```

Comment:
I think you should avoid importing from `__amalgam__` (explicitly). I think this is why you have a bunch of test failures